hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/domain/health/HealthCheckTaskTest.groovy

```diff
@@ -14,13 +16,18 @@ class HealthCheckTaskTest extends MultiZookeeperIntegrationTest {
     def modeService = new ModeService()
     ZookeeperClientManager manager
     HealthCheckTask healthCheckTask
+    def meterRegistry = Stub(MeterRegistry)
```

Comment:
could you try using `SimpleMeterRegistry` instead of these mocks?

Reply:
Done